src/interp/interp-wasm-c-api.cc

```diff
@@ -960,7 +960,8 @@ void wasm_global_set(wasm_global_t* global, const wasm_val_t* val) {
 own wasm_table_t* wasm_table_new(wasm_store_t* store,
                                  const wasm_tabletype_t* type,
                                  wasm_ref_t* init) {
-  return new wasm_table_t{Table::New(store->I, *type->As<TableType>())};
+  return new wasm_table_t{
+      Table::New(store->I, *type->As<TableType>(), Ref::Null)};
```

Comment:
It looks like `init` is ignored here.. is this something we should implement later?   Should we have a TODO here maybe?

Reply:
Nice catch! The API always provided an init value, but it was ignored before. Now we don't need to ignore it. This is kind of a bugfix then. I copied the code from `wasm_table_set`.